sensors/triggers/standard-k8s/standar-k8s.go

```diff
@@ -212,6 +212,22 @@ func (k8sTrigger *StandardK8sTrigger) Execute(ctx context.Context, events map[st
 
 		return k8sTrigger.namespableDynamicClient.Namespace(namespace).Patch(ctx, obj.GetName(), k8sTrigger.Trigger.Template.K8s.PatchStrategy, body, metav1.PatchOptions{})
 
+	case v1alpha1.Delete:
+		k8sTrigger.Logger.Info("deleting the object...")
+		_, err := k8sTrigger.namespableDynamicClient.Namespace(namespace).Get(ctx, obj.GetName(), metav1.GetOptions{})
+
+		if err != nil && apierrors.IsNotFound(err) {
+			k8sTrigger.Logger.Info("object not found, nothing to delete...")
```

Comment:
Need to return something here.

Reply:
I made it return nil, nil for no-op, which I'm pretty sure makes sense here